frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx

```diff
@@ -124,3 +124,13 @@ export function launchToYaml(cluster: Cluster, configMaps: ConfigMap[]) {
   const openShiftConsoleUrl = openShiftConsoleConfig?.data?.consoleURL
   window.open(`${openShiftConsoleUrl}/k8s/ns/${namespace}/hive.openshift.io~v1~${kind}/${name}/yaml`)
 }
+
+export function formatHiveLogText(message: string | undefined) {
+  if (typeof message === 'string') {
+    const messageArray = message.split('\n')
+    if (messageArray.length > 9) {
+      message = `${messageArray.slice(0, 8).join('\n')}...`
```

Comment:
There's a slight off-by-one error here. Other than that, it looks great. Good job.